probe/docker/container.go

```diff
@@ -277,12 +277,15 @@ func (c *container) ports(localAddrs []net.IP) report.StringSet {
 			continue
 		}
 		for _, b := range bindings {
-			if b.HostIP == "0.0.0.0" {
-				for _, ip := range localAddrs {
+			if b.HostIP != "0.0.0.0" {
```

Comment:
0.0.0.0 means any host address; hence if its not 0.0.0.0 we can use the address directly.  Otherwise we have to replace 0.0.0.0 with the host addresses.

Will add a comment,